SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1613,6 +1644,8 @@ void USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConne
 
 #endif // WITH_SERVER_CODE
 
+thread_local TArray<AActor*> GSenderStack;
```

Comment:
Is there a reason this isn't a member variable? In the editor I think all NetDrivers will run in serial on the same thread

Reply:
That might be true, but I do not want to take any risk.